<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeBattleReinforcementsFormClass</class>
 <widget class="QDialog" name="ChangeBattleReinforcementsFormClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>914</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChangeBattleReinforcementsForm</string>
  </property>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>914</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="1" column="4">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Army 1 reinforcement:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Army 2 reinforcement:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="3" alignment="Qt::AlignmentFlag::AlignHCenter">
     <widget class="QLabel" name="label_1">
      <property name="text">
       <string>Armies</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5" rowspan="6">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="ToMenu">
        <property name="styleSheet">
         <string notr="true">background-color:#F0FFFF;
font-weight:800;
color:black;
</string>
        </property>
        <property name="text">
         <string>Go back</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ApplyReinf">
        <property name="text">
         <string>Apply reinforcements</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="4">
     <widget class="QLineEdit" name="Army1Reinf"/>
    </item>
    <item row="1" column="0" rowspan="5" colspan="3">
     <widget class="QListWidget" name="ArmiesListWidget"/>
    </item>
    <item row="4" column="4">
     <widget class="QLineEdit" name="Army2Reinf"/>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Persent of army 1 reinforcement for 1 round</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Persent of army 2 reinforcement for 1 round</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLineEdit" name="Army1Percents"/>
    </item>
    <item row="4" column="3">
     <widget class="QLineEdit" name="Army2Percents"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
